website/docs/guides/cli.md

```diff
@@ -26,17 +26,16 @@ To install using [Homebrew](https://brew.sh) on macOS or Linux, run:
 
     $ brew install mcap
 
-### Using Go
+### From Source
 
-:::caution
-Installing via 'go install' is not recommended, and is not guaranteed to work.
-:::
+- :::caution
+- Installing via `go install` is not supported. To build from source you must clone the repository.
+- :::
```

Comment:
Sorry I think I was wrong in suggesting the `-`s - I got confused by the `-` in github indicating lines were deleted.

```suggestion
:::caution
Installing via `go install` is not supported. To build from source you must clone the repository.
:::
```

Reply:
Let's check the preview deploy whenever it is working